src/molecules/InputForm/InputForm.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Button from '../../atoms/Button/Button';
+import InputField from '../../atoms/InputField/InputField';
+
+import "./InputForm.css";
+
+const InputForm = ({
+    btnText, 
+    btnType, 
+    inputType, 
+    inputValue, 
+    onInputChange, 
+    onFormSubmit
+}) => {
+  
+    return (
+        <form onSubmit={onFormSubmit}>
+            <InputField inputType={inputType} inputValue={inputValue} onInputChange={onInputChange}/>
+            <Button btnType={btnType} btnText={btnText}/>
```

Comment:
Let pass onClick method to button only, and whenever it is clicked handle it accordingly from where it passed.